src/api.js

```diff
@@ -590,37 +130,10 @@ export {
   getDirectoryFile,
   setDirectoryFile,
```

Comment:
Saw that these were removed in your other PR

Reply:
Yup, at least `getDirectoryFile` is being used currently but I figured would be cleaner if I removed them in the other PR since they are related.